examples/particle-effects-demo/README.md

```diff
@@ -1 +1,3 @@
 A demo of various high quality particle effects (fire, magic, snow, rune spell...) that you can try and use in your game.
+
+The devil is in the detail and this little detail can often be found in particles! Explore the possibilities GDevelop offers to make a visually stunning game with the Particle Emitter object.
```

Comment:
"Devil is in the detail" the last "detail" should be plural!

I'd change "can often be found in" to "is often found in"